src/main/java/core/basesyntax/impl/WriteReportToFileImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.impl;
+
+import core.basesyntax.service.GenerateReport;
+import core.basesyntax.service.WriteReportToFile;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Map;
+
+public class WriteReportToFileImpl implements WriteReportToFile {
+    private static final String FILE_PATH = "src/main/java/core/basesyntax/report.csv";
+    private GenerateReport generateReport = new GenerateReportImpl();
```

Comment:
same with file path and this class has a dependency on another class (GenerateReport), let's fix it